docs/contributing.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to Manzan
+
+Thank you for considering contributing to **Manzan**! We welcome all contributions, from bug reports and documentation improvements to feature requests and code submissions.
+
+## How to Contribute
+
+### 1. Reporting Bugs
+
+If you find a bug in the project, please open an issue in the [GitHub Issues](https://github.com/ThePrez/Manzan/issues) section. Make sure to include:
+
+- A clear and descriptive title.
+- Steps to reproduce the issue.
+- The expected and actual behavior.
+- Any relevant logs, screenshots, or error messages.
+
+### 2. Suggesting Features or Enhancements
+
+We love new ideas! If you have a feature request or enhancement suggestion, please open an issue labeled as "enhancement" Describe the following:
+
+- The problem or limitation the feature would solve.
+- A detailed description of the proposed solution.
+- Alternatives you’ve considered, if applicable.
+
+### 3. Code Contributions
+
+We welcome pull requests (PRs) from everyone. To contribute code:
+
+1. **Fork the repository** and clone your fork to your machine.
+2. **Create a new branch** for your feature or bug fix:
+   ```bash
+   git checkout -b my-feature
+   ```
+3. **Make your changes** and commit them with clear, descriptive messages:
+    ```bash
+    git commit -m "Add feature XYZ"
+    ```
+4. **Push to your fork:**
+    ```bash
+    git push origin my-feature
+    ```
+5. **Submit a Pull Request** (PR) to the main branch of the main repository.
+
+Code Style
+Follow the existing code style used in the project.
+Ensure all new code is tested, documented, and free of linting issues.
```

Comment:
Not sure if this last part was intended to appear like this, but just added it as a step instead. Also, added a link to our CONTRIBUTING.md:

![image](https://github.com/user-attachments/assets/06810be0-f95d-43a0-9652-c072976622e4)

```suggestion
4. **Follow the existing code style** used in the project and ensure all new code is tested, documented, and free of linting issues.
5. **Push to your fork:**
    ```bash
    git push origin my-feature
    ```
6. **Submit a Pull Request** (PR) to the main branch of the main repository.

For more instructions on how to setup your development environment, build the Mazan components (handler/distributor), and run tests, check out the detailed guide [here](https://github.com/ThePrez/Manzan/blob/main/CONTRIBUTING.md).
```